package.json

```diff
@@ -8,11 +8,19 @@
   "scripts": {
     "storybook": "start-storybook"
   },
+  "jest": {
+    "testRegex": [
+      "spec/javascripts/.*_test\\.js"
+    ]
+  },
   "devDependencies": {
     "@storybook/addon-controls": "^6.3.7",
     "@storybook/addon-docs": "^6.3.7",
     "@storybook/server": "^6.3.7",
+    "babel-jest": "^27.0.6",
```

Comment:
It looks like we don't need to add this as an explicit dev dependency, as it's already required by `@jest-core`.

Reply:
Right 👌 Thanks!